cybathlon_feedback:
    Wheelchair:
        soft_threshold_left:  0.75
        soft_threshold_right:  0.75
        hard_threshold_left:  0.9
        hard_threshold_right:  0.9
        reset_on_soft: false
        reset_on_hard: false

    ScreenCursor:
        soft_threshold_left:  0.6
        soft_threshold_right:  0.6
        hard_threshold_left:  0.8
        hard_threshold_right:  0.8
        reset_on_soft: true
        reset_on_hard: false

    RoboticArm:
        soft_threshold_left:  0.8
        soft_threshold_right:  0.8
        hard_threshold_left:  1.0
        hard_threshold_right:  1.0
        reset_on_soft: false
        reset_on_hard: true
